src/cheri/rvy32-encoding.adoc

```diff
@@ -218,13 +218,13 @@ NOTE: Software is completely free to define the usage of these bits.
 [#section_cap_sealed32]
 ==== Capability Type (CT) Encoding
 
-Capabilities in this encoding have a 1-bit field for <<sec_cap_type>> values which behaves in the same way as the {rvy64_uni_base_name} <<section_cap_sealed>>.
+Capabilities in this encoding have a 1-bit field for <<sec_cap_type>> values which behaves in the same way as the <<section_cap_sealed,{rvy64_uni_base_name}>>.
```

Comment:
This one was actually intentional 'as the RVY64_L... capability type (T) encoding'

Reply:
reverted